modules/odc-installing-helm-charts.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// applications/application_life_cycle_management/odc-working-with-helm-charts-using-developer-perspective.adoc
+
+[id="odc-installing-helm-charts_{context}"]
+= Installing Helm Charts
+
+To install a Helm chart or create a Helm release:
```

Comment:
Just a suggestion - I think this can be reworded to highlight what is being done in the procedure below. Something like - You can install Helm chart, or create a Helm release using the Developer Console. 

Also, are there any pre-reqs that should be added here, like Admin access to Dev Console ( I am not sure)

Reply:
I see your have already added this info in odc-working-with-helm-charts-using-developer-perspective.adoc, and the install file is part of that assembly. 

I think in that case, you can simply remove the sentence "To install a ...", as the heading is self explanatory

Reply:
Cool, sounds good. I am not very satisfied with the structure, currently. Need to do this to bypass the no links restrictions in modules. Let me try to work around it. Thank you!